OpenDuetPkg.dsc

```diff
@@ -97,11 +97,18 @@
   HobLib|MdePkg/Library/DxeHobLib/DxeHobLib.inf
   CpuExceptionHandlerLib|MdeModulePkg/Library/CpuExceptionHandlerLibNull/CpuExceptionHandlerLibNull.inf
   ExtractGuidedSectionLib|MdePkg/Library/DxeExtractGuidedSectionLib/DxeExtractGuidedSectionLib.inf
+  OcDebugLogLib|OpenCorePkg/Library/OcDebugLogLibNull/OcDebugLogLibNull.inf
```

Comment:
Do not think we need OcDebugLogLib. Just DebugLib will be enough throughout the code.